distribution/ddf-common/src/main/resources/bin/ddf

```diff
@@ -1,23 +1,199 @@
 #!/bin/sh
 
-SCRIPTDIR=$(dirname "$0")
+# CONFIGURATION OPTIONS
```

Comment:
This script was changed to support  DDF's "unzip and run" design goal. The changes to the script are to start/stop the Solr server.

This script has some configurable options:  enable HTTPS and require TLS client authentication. In retrospect, this is probably gold plating and unnecessary. In the "internally managed" configuration, this script starts a Solr server on the same host as DDF and configures the Solr server to request all remote requests. For such a configuration, TLS should not be necessary